assets/styles/jon.scss

```diff
@@ -0,0 +1,8 @@
+#test{
+  .name1{
+    border: 1px solid #000;
+    display: none;
+    float: left;
+    color: blue;
```

Comment:
Color `blue` should be written in hexadecimal form as `#0000ff`<br>Color literals like `blue` should only be used in variable declarations; they should be referred to via variable everywhere else.